src/clients/python/experimental_api_v2/library/httpclient.py

```diff
@@ -1120,6 +1120,11 @@ def async_infer(self,
             Optional url query parameters to use in network
             transaction.
 
+        Returns
+        -------
+        Greenlet object
```

Comment:
Period at end. Is there a link to what a Greenlet is that you could add here?

Reply:
Added the link.